.github/workflows/ci.yml

```
@@ -139,6 +149,24 @@ jobs:
     - run: yarn --frozen-lockfile
 
     - run: sudo apt-get install xvfb
-
     - name: Run smoke tests
       run: xvfb-run --auto-servernum yarn smoke --debug -j=1 --retries=2 --invert-match ${{ matrix.smoke-test-invert }} $SMOKE_GROUP_1
+  
+  # Only run smoke tests for windows against stable chrome.
+  smoke-windows:
+    runs-on: windows-latest
+    name: smoke_stable_windows
+
+    steps:
+    - name: git clone
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 10.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+
+    - run: yarn --frozen-lockfile
+
+    - name: Run smoke tests
+      run: yarn smoke --debug -j=1 --retries=2 dbw a11y oopif redirects seo offline lantern metrics
```

from meeting discussion

```suggestion
      run: yarn smoke --debug -j=1 --retries=2 dbw oopif offline lantern metrics
```